Content/task-estimation.md

```diff
@@ -0,0 +1,68 @@
+# Task estimation in SCRUM: 
+
+![picture alt](https://focus.independent.ie/thumbor/k2_DlYS8OJVEETmHVusgZUw0ZTw=/0x1114:1178x1764/1280x853/prod-mh-ireland/60f6ec3e-c1e5-11ed-8014-0210609a3fe2.jpg "SCRUM!")
+
+Sprint planning meeting. Outline tasks to be completed during sprint. 
+
+Weighting sprint tasks. Team-wide estimation of task length and difficulty. 
+
+Assign tasks to team members. 
+
+Start the sprint, do the work. 
+
+Daily stand-ups with SCRUM master. What have you done? What will you do? Any blockers? 
+
+Sprint demo meeting, showcase work done during sprint. 
+
+Sprint retro, what went well, what can be improved, any comments. 
+
+  
+
+Notes: 
+
+Story point: A story point is a unit of measurement for a user story based on factors such as complexity, effort, uncertainty, and risk.  
+
+Epic: Epics serve as large bodies of work that can be broken down into smaller and manageable tasks. 
+
```

Comment:
No need for the notes - we can remove this.

Reply:
Agreed